modules/UI/videolayout/Filmstrip.js

```diff
@@ -153,7 +153,11 @@ const Filmstrip = {
             return;
         }
         if (sendAnalytics) {
-            sendAnalyticsEvent(TOOLBAR_FILMSTRIP_TOGGLED);
+            // TODO: This should be converted to an event which indicates
+            // why/how the filmstrip was toggled (i.e. was it a button click,
+            // keyboard shortcut, API call).
```

Comment:
Do you also find it odd that a parameter was added not to send analytics?

Reply:
Yes :(
Thinking about this some more, I'll just remove this event (and the parameter). I already added an event when this happens as a result of an API call, IIRC there is a UI event for the keyboard shortcut, and I'll see if I can add one for the button being clicked.